src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesGcm.OpenSsl.cs

```diff
@@ -10,12 +10,14 @@ namespace System.Security.Cryptography
     public sealed partial class AesGcm
     {
         private SafeEvpCipherCtxHandle _ctxHandle;
+        private static readonly KeySizes s_tagByteSizes = new KeySizes(12, 16, 1);
+        private static readonly bool s_isSupported = Interop.OpenSslNoInit.OpenSslIsAvailable;
 
-        public static bool IsSupported { get; } = Interop.OpenSslNoInit.OpenSslIsAvailable;
-        public static KeySizes TagByteSizes { get; } = new KeySizes(12, 16, 1);
+        public static partial bool IsSupported => s_isSupported;
```

Comment:
The downside to the `partial` trick for documentation purposes is that partial properties cannot be auto-properties. This seemed like a worthy tradeoff (and I am not sure how to solve it otherwise).